auth_otp/services/otp_service.py

```
@@ -0,0 +1,341 @@
+            user_agent
+        )
+    
+    def _check_rate_limit(self, phone_number: str) -> Tuple[bool, str]:
+        """بررسی محدودیت نرخ"""
+        rate_limit, created = OTPRateLimit.objects.get_or_create(
+            phone_number=phone_number
+        )
+        
+        return rate_limit.can_send_otp()
+    
+    def _update_rate_limit(self, phone_number: str):
```

Ruff (0.12.2)

Missing return type annotation for private function `_update_rate_limit`

Add return type annotation: `None`

(ANN202)